app/models/frontend/nics/ExemptionReason.scala

```diff
@@ -33,4 +34,12 @@ object ExemptionReason extends Enum[ExemptionReason] with utils.PlayJsonEnum[Exe
     override val exemptionCode: String = "003"
   }
 
+  def fromNonFinancialType(exemption: AnnualNonFinancialsType.Class4NicsExemptionReason.Value): ExemptionReason =
+    exemption match {
+      case AnnualNonFinancialsType.Class4NicsExemptionReason._003 =>
+        ExemptionReason.DiverDivingInstructor
+      case AnnualNonFinancialsType.Class4NicsExemptionReason._002 =>
+        ExemptionReason.TrusteeExecutorAdmin
```

Comment:
Shall we match on the other `Class4NicsExemptionReason` (_001, _004, _005, _006) or have a `case _` ?

Reply:
The other exemption reasons are not used in this service